Exiled.API/Features/Player.cs

```diff
@@ -381,13 +381,7 @@ public bool IsOverwatchEnabled
         public bool IsNoclipPermitted
         {
             get => FpcNoclip.IsPermitted(ReferenceHub);
-            set
-            {
-                if (value && !FpcNoclip.PermittedPlayers.Contains(ReferenceHub.netId))
-                    FpcNoclip.PermitPlayer(ReferenceHub);
-                else if (!value && FpcNoclip.PermittedPlayers.Contains(ReferenceHub.netId))
-                    FpcNoclip.UnpermitPlayer(ReferenceHub);
-            }
+            set => _ = value ? FpcNoclip.PermitPlayer(ReferenceHub) : FpcNoclip.UnpermitPlayer(ReferenceHub)
```

Comment:
```suggestion
            set => value ? FpcNoclip.PermitPlayer(ReferenceHub) : FpcNoclip.UnpermitPlayer(ReferenceHub)
```

Reply:
It wont work compile without `_ =`, C# operators moment

Reply:
Yes, because you can't assign a void value to the discard operator `_`.

Reply:
the problem is from a missing ;